functions/src/api/v1/notification/sendTopicNotification.ts

```diff
@@ -1,26 +0,0 @@
-import { Request, Response } from 'express'
-import * as admin from 'firebase-admin'
-
-export const sendTopicNotification = async (req: Request, res: Response) => {
```

Comment:
I think this made it easier to test notifications locally when developing for mobile devices. Otherwise I need to make sure I'm using the correct user, who has permissions, that the group I want to subscribe to exists ...

Reply:
it may make it easier but does not test the whole scope! I can make some api helpers to populate the db with specific datas.

Reply:
yes but for instance
- I have to attach a sound to a notification -> I only need to receive one -> no need to test the whole flow
- I have to make the notification persistent (to simulate a call) -> no need to test the whole flow

Reply:
sendTestPushNotification with `/api/v1-tests/notification/push`